src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -207,10 +211,10 @@ define(function (require, exports, module) {
         var currentPath = path || (currentDoc && currentDoc.doc.file.fullPath);
         var escapedPath = escapedPathTemplate({path: currentPath});
 
-        return '<base href="' + window.location.href + '">\n' +
+        return '<base href="' + UrlCache.getBaseUrl() + '">\n' +
             "<script>\n" + PostMessageTransportRemote + "</script>\n" +
-            "<script>\n" + XHRShim + "</script>\n" +
-            MouseManager.getRemoteScript(escapedPath) +
+            XHRManager.getRemoteScript() +
+            MouseManager.getRemoteScript(currentPath) +
```

Comment:
Good catch, this was reverted accidentally in a rebase.  See why I need/want you reviewing my crappy code?  :)